File: tatala-java/src/com/qileyuan/tatala/executor/SocketServerTarget.java
package com.qileyuan.tatala.executor;

import com.qileyuan.tatala.socket.client.SocketController;
import com.qileyuan.tatala.socket.to.TransferObject;

/**
 * This class is socket server target, provider socket distribution-method-call.
 * @author JimT
 *
 */
public class SocketServerTarget implements ServerTarget{
	public Object execute(TransferObject to) {
		return SocketController.execute(to);
	}
}

File: history/0.1.2/tatala-client-csharp/src/QiLeYuan/Tatala/executor/SocketServerTarget.cs
using System;
using QiLeYuan.Tatala.socket.client;
using QiLeYuan.Tatala.socket.to;

/**
 * This class is socket server target, provider socket distribution-method-call.
 * @author JimT
 *
 */
namespace QiLeYuan.Tatala.executor {

    public class SocketServerTarget : ServerTarget {

        public Object execute(TransferObject to) {
            Object resultObject = null;

            resultObject = SocketController.execute(to);

            return resultObject;
        }
    }
}